docs/time/index.rst

```diff
@@ -264,7 +264,7 @@ both" [#]_. See also [#]_ and [#]_.
 ::
 
   >>> Time.SCALES
-  ('tai', 'tcb', 'tcg', 'tdb', 'tt', 'ut1', 'utc')
+  ('tai', 'tcb', 'tcg', 'tdb', 'tt', 'ut1', 'utc', 'local')
```

Comment:
You should also adjust the text so that it becomes clear what 'local' is, maybe including the table below.